build.sbt

```diff
@@ -8,6 +8,7 @@ val `sbt-scalajs-bundler` =
     .settings(commonSettings)
     .settings(
       sbtPlugin := true,
+      version := "0.11.0",
```

Comment:
Not sure how @julienrf manages versions, but this should probably not be here.

Reply:
Yes, I have to correct this, it was for my tests.